encoder-brave/src/main/resources/META-INF/native-image/io.zipkin.contrib.otel/encoder-brave/resource-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "scope\\.properties"
+      }
+    ]
+  }
+}
```

Comment:
supernit.. a lot of your files are missing EOL before EOF, and seeing the red minus is distracting ;) maybe there's an editorfile about this, I forget.

Reply:
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/67f89912d618cfaafc14a20f542b2bd8dba8898e
I've never cared about that